backend/apps/users/signals.py

```diff
@@ -0,0 +1,16 @@
+from django.db.models.signals import post_save
+from django.dispatch import receiver
+from apps.notificationpref.models import UserNotificationPreference
+from apps.users.models import BaseUser
+
+@receiver(post_save, sender=BaseUser)
```

Comment:
What's the purpose of this? Is it to create a UserNotificationsPreference when a BaseUser is created? If so, move this to the user_create service in (/users/services.py) as other logic like sending confirmation emails goes there too, and it's a lot easier to understand what's happening that way.

Reply:
Inside user_create (users/services.py) you can add a call to a notifications_preference_create function and pass the user. Just the same as the confirmation email create